pkg/common/configs/configvalidator.go

```diff
@@ -86,6 +86,14 @@ type placementStaticPath struct {
 
 type placementPathCheckResult int
 
+type LimitCheckHelper struct {
```

Comment:
LimitValidator? or something like that

Reply:
Good suggestion, addressed in latest PR.